gammapy/scripts/download.py

```diff
@@ -0,0 +1,107 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Command line tool to download datasets and notebooks from gammapy-extra GitHub repo."""
+from __future__ import absolute_import, division, print_function, unicode_literals
+import logging
+import click
+
+import sys
+import requests
+import json
+from ..extern.pathlib import Path
+from ..extern.six.moves.urllib.request import urlretrieve
+
+log = logging.getLogger(__name__)
+
+apigitUrl = 'https://api.github.com/repos/gammapy/gammapy-extra/git/trees/master:'
+rawgitUrl = 'https://raw.githubusercontent.com/gammapy/gammapy-extra/master/'
+localfolder = Path('./gammapy-extra')
```

Comment:
Expose this folder where files are put as a command line option (either on the `gammapy download` click group, or on the `notebooks` and `datasets` click commands?

I'm not a fan of the default name `gammapy-extra`. We will fetch notebooks and data from other places, i.e. get away from gammapy-extra a bit, no?

Not sure what to suggest here: there needs to be a decision whether notebooks and data have to be next to each other or can be separate. If they have to be in the same folder, then we could use `../datasets/something` from the tutorial notebooks, if it can be separate we'd use an env var to access data files, probably called `GAMMAPY_DATA`.